### PR TITLE
Product interview sidebar

### PR DESCRIPTION
**Pull request by Zacharis278**
### [MST-1703](https://2u-internal.atlassian.net/browse/MST-1703)

Add widget for scheduling product team interviews

Screen Shot 2022-11-28 at 9 30 44 AM

### CODE REVIEW

**Review by alangsto — APPROVED** [review]
👍

**Review by mattcarter — APPROVED** [review]
approved

File: src/containers/WidgetContainers/LoadedSidebar/index.jsx [hunk 1]
```diff
@@ -1,9 +1,13 @@
 import React from 'react';
 
+import InterviewWidget from 'widgets/InterviewWidget';
 import RecommendationsPanel from 'widgets/RecommendationsPanel';
 
 export const WidgetSidebar = () => (
   <div className="widget-sidebar">
```
Comment (wittjeff) [inline src/containers/WidgetContainers/LoadedSidebar/index.jsx:7]: Can you make this an <aside>?

**Review by wittjeff — CHANGES_REQUESTED** [review]

**Comment by justinhynes** [comment]
Hi @Zacharis278!

I'm trying to check up on some open PRs for this repo. Is this change still needed? If not, may I close it? If so, could you address the open requests and fix the conflicts on the PR? 

Thanks in advance!

**Comment by Zacharis278** [comment]
@justinhynes releasing this was put on hold by the product team so it's fallen way behind main. I'll have to check in on if/when this can actually be merged and get back to you.

**Comment by justinhynes** [comment]
Considering this is over a year old without much movement, I'm going to close this PR. Please resubmit a PR if this becomes a priority again. Thanks!

**Pull request closed without merging**